src/python/bot/fuzzers/syzkaller/config.py

```diff
@@ -35,24 +35,24 @@ def run(serial,
 
   """
   devices = {}
-  devices["devices"] = [serial]
+  devices['devices'] = [serial]
   data = {}
   data['target'] = 'linux/arm64'
-  data["reproduce"] = reproduce
-  data["workdir"] = work_dir_path
-  data["http"] = "localhost:50001"
-  data["syzkaller"] = binary_path
-  data["suppressions"] = ["do_rt_sigqueueinfo", "do_rt_tgsigqueueinfo"]
-  data["vm"] = devices
-  data["kernel_obj"] = vmlinux_path
-  data["sandbox"] = "android"
-  data["ignores"] = ["WARNING:", "INFO:"]
-  data["type"] = "adb"
-  data["procs"] = 1
-  data["cover"] = kcov
+  data['reproduce'] = reproduce
+  data['workdir'] = work_dir_path
+  data['http'] = 'localhost:50001'
+  data['syzkaller'] = binary_path
+  data['suppressions'] = ['do_rt_sigqueueinfo', 'do_rt_tgsigqueueinfo']
```

Comment:
We should consider what this should be for android (its fine is its in a different cl)

Reply:
sg, will add a todo for this purpose.